### PR TITLE
847: Add empty version of 12-week correspondence page.

### PR DESCRIPTION
**Pull request by ahernp**
Trello card [#847](https://trello.com/c/zQ3ZsnZ0/847-create-void-page-on-12-week-correspondence-which-suggests-moving-to-the-next-page-design-attached).

### CODE REVIEW

File: accessibility_monitoring_platform/settings/base.py [hunk 1]
```diff
@@ -237,6 +237,6 @@
 
 CSP_DEFAULT_SRC = ("'none'",)
 CSP_STYLE_SRC = ("'self'", "'unsafe-inline'")
-CSP_SCRIPT_SRC = ("'self'", )
-CSP_FONT_SRC = ("'self'")
+CSP_SCRIPT_SRC = ("'self'",)
+CSP_FONT_SRC = "'self'"
```
Comment (richardbored) [inline accessibility_monitoring_platform/settings/base.py:241]: This needs to be in a tuple or list

File: report_viewer/settings/base.py [hunk 2]
```diff
@@ -220,7 +220,7 @@
 }
 
 CSP_DEFAULT_SRC = ("'none'",)
-CSP_STYLE_SRC = ("'self'")
+CSP_STYLE_SRC = "'self'"
 CSP_SCRIPT_SRC = ("'self'",)
-CSP_FONT_SRC = ("'self'")
+CSP_FONT_SRC = "'self'"
```
Comment (richardbored) [inline report_viewer/settings/base.py:225]: Tuple or list

**Review by richardbored — CHANGES_REQUESTED** [review]

File: accessibility_monitoring_platform/settings/base.py (hunk 1 above)
Comment (ahernp) [reply accessibility_monitoring_platform/settings/base.py]: I don't know were there changes came from.

File: report_viewer/settings/base.py (hunk 2 above)
Comment (ahernp) [reply report_viewer/settings/base.py]: I don't know were these changes came from.

Comment (ahernp) [reply report_viewer/settings/base.py]: They have been undone.

File: accessibility_monitoring_platform/settings/base.py (hunk 1 above)
Comment (ahernp) [reply accessibility_monitoring_platform/settings/base.py]: They have been (mostly) undone.

**Review by richardbored — APPROVED** [review]

**Pull request merged**